lib/get-exporter.js

```diff
@@ -0,0 +1,116 @@
+const paginateRequest = require('./paginate-request');
+const inquirer = require('inquirer');
+const {
+  loadConfig,
+  getShallowStatus,
+  logResult,
+  logProcessing,
+  logError,
+  mapById,
+  plural
+} = require('./utils');
+const {
+  remove,
+  writeFile,
+  ensureDir
+} = require('fs-extra');
+const { merge, values, isEmpty } = require('omnibelt');
+const allSettledSerial = require('./all-settled-serial-p');
+const { rollbarLog } = require('./rollbar');
+
+const getExporter = ({
+  apiType, commandType, localStatusParams, remoteStatusParams, getData, extraQuery
+}) => {
+  const exporter = async (pattern, command = {}, loadedConfig) => {
+    await ensureDir(apiType);
+    let didExport = false;
+    const { apiToken, applicationId, api } = loadedConfig ? loadedConfig : await loadConfig();
+    if (!apiToken || !applicationId) { return; }
+    let items;
+    try {
+      let query = { applicationId };
+      if (extraQuery) {
+        query = merge(query, extraQuery);
+      }
```

Comment:
I would remove this if statement since none of the exported resources need it right now.